### PR TITLE
Solved JS Assignments 

### PR DESCRIPTION
**Pull request by Nimaxor**

### CODE REVIEW

**Comment by BabyElias** [comment]
Good efforts!
However, please make a commit with at least 140 score to pass the assignment.

File: src/conditionalAndLoops.js [hunk 1]
```diff
@@ -19,8 +19,20 @@
...
+    return "Fizz"
+  }
+  else if (num % 5 ==0) {
+    return "Buzz"
+  }
+  else {
+    return num
+  }	
 }
+getFizzBuzz(15);
```
Comment (EshaanAgg) [inline src/conditionalAndLoops.js]: Remove or comment out these calls to functions. They interfere with the testing script.

**Review by EshaanAgg — CHANGES_REQUESTED** [review]
Please make these changes.

File: src/array.js [hunk 2]
```diff
@@ -42,8 +54,16 @@ function doubleArray(arr) {
...
-	throw new Error("Not implemented");
+  let posarr=[];
+	for (let i=0;i < arr.length ; i++ ) {
+    if (arr[i]>0) {
+      posarr.push(arr[i]);
+    }
+  }
+  return posarr ;
 }
+console.log(getArrayOfPositives([ 0, 1, 2, 3, 4, 5 ]));
```
Comment (EshaanAgg) [inline src/array.js]: Delete all these `console.log`s from all the test files as well. They can cause the test suite to break undetermistically, and clog up the testing window screen.

File: src/conditionalAndLoops.js [hunk 3]
```diff
@@ -208,7 +208,9 @@ function isBracketsBalanced(str) {
  *   Date('2000-01-01 01:00:00.100'), Date('2015-01-02 03:00:05.000')  => '15 years ago'
  *
  */
-
+function timespanToHumanString(startDate, endDate) {
```
Comment (vaibhav-1508) [inline src/conditionalAndLoops.js:211]: test suite was unable to run cuz of the missing function

**Pull request closed without merging**